### PR TITLE
NOISSUE - additional test

### PR DESCRIPTION
**Pull request by mteodor**
Signed-off-by: Mirko Teodorovic <mirko.teodorovic@gmail.com>

### CODE REVIEW

**Review by drasko — CHANGES_REQUESTED** [review]
This test needs documentation.

**Review by drasko — COMMENTED** [review]
@blokovi @dusanb94 we are missing reports for this one.

I would accept it for now, and then @blokovi can add benchmarks.

Opinions?

File: mzbench/scenarios/burnout.bdl [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        "MF_MZBENCH_MSG_SIZE" = 100,
+        "MF_MZBENCH_QOS" = 2,
+        "MF_MZBENCH_MQTT_ENDPOINT" = "127.0.0.1",
+        "MF_MZBENCH_MQTT_PORT" = 1883
+)
+
+make_install(git = "https://github.com/erlio/vmq_mzbench.git",
+             branch = "master")
+
+
```
Comment (drasko) [inline mzbench/scenarios/burnout.bdl]: Kill this white line

**Review by drasko — CHANGES_REQUESTED** [review]

File: mzbench/README.md [hunk 2]
```diff
@@ -77,3 +78,6 @@ A prerequisite for running 1-to-1 test scenarios is to have Mainflux instance wh
 | MF_MZBENCH_PUB_TIME      | Duration of publishing messages in minutes                                                       | 5         |
 | MF_MZBENCH_MSG_SIZE      | Size of messages in bytes. Messages are a sequence of random bytes                               | 100       |
 | MF_MZBENCH_QOS           | MQTT QoS level                                                                                   | 2         |
+
+## Burnout 
+Burnout test is similar to the `Fan-In` just that message rate function is `comb` not a constant. Test is intended to be run for a long period of time to detect possible resource problem in deployment. By default it runs for 300 min, publishing for 1 min and then pause for 9 min.
```
Comment (dborovcanin) [inline mzbench/README.md:83]: Maybe we should switch the ratio to 2:8? 1:9 sounds like a lot of waiting and not enough load.

File: mzbench/scenarios/burnout.bdl [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
```
Comment (blokovi) [inline mzbench/scenarios/burnout.bdl:2]: Scenario must start with `#!benchDL`. Without it, you get parse error.

```MZBench supports two languages: eDSL based on erlang terms (default) and benchDL, to let it know that you use second option you need to specify this at the very beginning```

**Review by blokovi — CHANGES_REQUESTED** [review]

**Review by blokovi — APPROVED** [review]

File: mzbench/scenarios/burnout.bdl (hunk 3 above)
Comment (drasko) [reply mzbench/scenarios/burnout.bdl:2]: @blokovi is this OK now?

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**